bundles/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/handler/TeslaVehicleHandler.java

```diff
@@ -263,6 +265,18 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                             }
                             break;
                         }
+                        case CHARGE_AMPS:
+                            if (command instanceof QuantityType) {
```

Comment:
You could also accept Decimal type?

Reply:
I guess so, yes. I've changed the code accordingly.